### PR TITLE
prl-citizen-frontend set up for aat and demo

### PR DESCRIPTION
**Pull request by AlokDatta**
### Change description ###
flux-setup for prl-citizen-frontend for AAT and DEMO environment

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by PraveenGuntoju — APPROVED** [review]

File: k8s/namespaces/private-law/prl-citizen-frontend/prl-citizen-frontend.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/prl-citizen-frontend
+  values:
+    java:
+      replicas: 2
+      memoryRequests: "768Mi"
+      cpuRequests: "1000m"
+      memoryLimits: "3072Mi"
+      cpuLimits: "2500m"
```
Comment (timja) [inline k8s/namespaces/private-law/prl-citizen-frontend/prl-citizen-frontend.yaml:21]: do you need all these resources? this is quite heavy for a new app

File: k8s/namespaces/private-law/prl-citizen-frontend/prl-citizen-frontend.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      readinessDelay: 45
+      readinessTimeout: 5
+      readinessPeriod: 15
```
Comment (timja) [inline k8s/namespaces/private-law/prl-citizen-frontend/prl-citizen-frontend.yaml:24]: is there something wrong with the default? new apps should generally work OOTB

File: k8s/namespaces/private-law/prl-citizen-frontend/demo.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/prl-citizen-frontend
+  values:
+    java:
+      replicas: 2
```
Comment (timja) [inline k8s/namespaces/private-law/prl-citizen-frontend/demo.yaml:17]: replicas should be set in the base file not env specific unless you're overriding

File: k8s/namespaces/private-law/prl-citizen-frontend/demo.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      memoryRequests: "768Mi"
+      cpuRequests: "1000m"
+      memoryLimits: "3072Mi"
+      cpuLimits: "2500m"
+      readinessDelay: 45
+      readinessTimeout: 5
+      readinessPeriod: 10
+      useInterpodAntiAffinity: true
```
Comment (timja) [inline k8s/namespaces/private-law/prl-citizen-frontend/demo.yaml:25]: don't duplicate config

**Review by timja — CHANGES_REQUESTED** [review]
seems a lot of duplication, can you please clean it up?

File: k8s/namespaces/private-law/prl-citizen-frontend/prl-citizen-frontend.yaml (hunk 1 above)
Comment (AlokDatta) [reply k8s/namespaces/private-law/prl-citizen-frontend/prl-citizen-frontend.yaml]: thanks @timja , removed.

File: k8s/namespaces/private-law/prl-citizen-frontend/prl-citizen-frontend.yaml (hunk 2 above)
Comment (AlokDatta) [reply k8s/namespaces/private-law/prl-citizen-frontend/prl-citizen-frontend.yaml]: addressed and removed. default will be followed

File: k8s/namespaces/private-law/prl-citizen-frontend/demo.yaml (hunk 3 above)
Comment (AlokDatta) [reply k8s/namespaces/private-law/prl-citizen-frontend/demo.yaml]: base file contains the replicas, so removed duplicate config

File: k8s/namespaces/private-law/prl-citizen-frontend/demo.yaml (hunk 4 above)
Comment (AlokDatta) [reply k8s/namespaces/private-law/prl-citizen-frontend/demo.yaml]: removed duplicate config

**Review by timja — APPROVED** [review]

**Review by sairamkantheti1 — APPROVED** [review]

**Pull request merged**